frontend/src/component/FeedItem/FeedItemHeader.jsx

```diff
@@ -64,6 +68,11 @@ export class FeedItemHeader extends React.Component {
       props.content.currentRevisionType &&
       props.lastModifier
     )
+    const shouldShowChangeTypeButton = findUserRoleIdInWorkspace(
+      props.user.userId,
+      (props.workspaceList.find(workspace => workspace.id === props.workspaceId) || {}).memberList || [],
+      ROLE_LIST
+    ) >= ROLE.contentManager.id && props.content.contentNamespace === CONTENT_NAMESPACE.PUBLICATION && props.onClickChangeType !== undefined
```

Comment:
props.onClickChangeType !== undefined

might require to be updated depending on default value of props.onClickChangeType

Reply:
done